src/components/CompareResults/ResultsTable.tsx

```diff
@@ -85,22 +78,17 @@ function ResultsTable(props: { results: CompareResultsItem[][] }) {
   // TODO Implement a loading UI through the react-router defer mechanism
   const loading = false;
 
-  const themeColor100 =
-    themeMode === 'light' ? Colors.Background100 : Colors.Background100Dark;
-
   const styles = {
     tableContainer: style({
-      backgroundColor: themeColor100,
```

Comment:
this backgroundColor was clashing with the `background: none` above. Instead I removed both of them to keep it transparnet, so that we see the one from ResultsMain instead.